rijkshuisstijl/sass/components/form/_form.scss

```diff
@@ -104,4 +105,9 @@
   &__actions .button:not(:last-child) {
     margin-right: $form-control-spacing;
   }
+
+  &__description {
+    @include text;
+    padding: $description-padding-h 0;
```

Comment:
waar komt deze constante vandaan? Het eerste woord in de naam geeft overigens aan uit welk component/library moet komen. Het lijkt mij (afhankelijk van waar het vandaan komt) dat het iets als `$form-description-padding-h` wordt.